ytmusicapi/parsers/_utils.py

```diff
@@ -67,9 +67,12 @@ def get_fixed_column_item(item: JsonDict, index: int) -> JsonDict | None:
     return typing.cast(JsonDict, item["fixedColumns"][index]["musicResponsiveListItemFixedColumnRenderer"])
 
 
+DOT_SEPARATOR_RUN = {"text": " • "}
```

Comment:
I think it's a bit unexpected to place this in the middle of `_utils`.

If you want to keep it please create a separate file `parsers/constants.py`.